src/pages/iou/SplitBillDetailsPage.tsx

```diff
@@ -0,0 +1,169 @@
+        [reportID, reportAction, draftTransaction, session?.accountID, session?.email],
+    );
+
+    return (
+        <ScreenWrapper testID={SplitBillDetailsPage.displayName}>
+            <FullPageNotFoundView shouldShow={isEmptyObject(reportID) || isEmptyObject(reportAction) || isEmptyObject(transaction)}>
+                <HeaderWithBackButton title={translate('common.details')} />
+                <View style={[styles.containerWithSpaceBetween, styles.pointerEventsBoxNone]}>
+                    {isScanning && (
+                        <MoneyRequestHeaderStatusBar
+                            title={translate('iou.receiptStatusTitle')}
+                            description={translate('iou.receiptStatusText')}
+                            shouldShowBorderBottom
+                        />
+                    )}
+                    {Boolean(participants.length) && (
+                        <MoneyRequestConfirmationList
+                            hasMultipleParticipants
+                            payeePersonalDetails={payeePersonalDetails}
+                            selectedParticipants={participantsExcludingPayee}
+                            iouAmount={splitAmount}
+                            iouCurrencyCode={splitCurrency}
+                            iouComment={splitComment}
+                            iouCreated={splitCreated}
+                            iouMerchant={splitMerchant}
+                            iouCategory={splitCategory}
+                            iouTag={splitTag}
+                            iouIsBillable={splitBillable}
+                            iouType={CONST.IOU.TYPE.SPLIT}
+                            isReadOnly={!isEditingSplitBill}
+                            shouldShowSmartScanFields
+                            receiptPath={transaction?.receipt?.source}
+                            receiptFilename={transaction?.filename}
+                            shouldShowFooter={false}
+                            isEditingSplitBill={isEditingSplitBill}
+                            hasSmartScanFailed={hasSmartScanFailed}
+                            reportID={reportID}
+                            reportActionID={reportAction?.reportActionID}
+                            transaction={isEditingSplitBill ? draftTransaction || transaction : transaction}
+                            onConfirm={onConfirm}
+                            isPolicyExpenseChat={ReportUtils.isPolicyExpenseChat(report)}
+                            policyID={ReportUtils.isPolicyExpenseChat(report) ? report?.policyID : null}
+                        />
+                    )}
+                </View>
+            </FullPageNotFoundView>
+        </ScreenWrapper>
+    );
+}
+
+SplitBillDetailsPage.displayName = 'SplitBillDetailsPage';
+
+const WrappedComponent = withOnyx<SplitBillDetailsPageProps, Pick<SplitBillDetailsPageOnyxProps, 'transaction' | 'draftTransaction'>>({
```

Comment:
Maybe let's split types a little to make withOnyx HOC typing more clear
```
type SplitBillDetailsPageTransactionOnyxProps = {
    /** The current transaction */
    transaction: OnyxEntry<Transaction>;

    /** The draft transaction that holds data to be persisited on the current transaction */
    draftTransaction: OnyxEntry<Transaction>;
};

type SplitBillDetailsPageOnyxPropsWithoutTransaction = {
    /** The personal details of the person who is logged in */
    personalDetails: OnyxEntry<PersonalDetailsList>;

    /** The active report */
    report: OnyxEntry<Report>;

    /** Array of report actions for this report */
    reportActions: OnyxEntry<ReportActions>;

    /** Session info for the currently logged in user. */
    session: OnyxEntry<Session>;
};

type SplitBillDetailsPageOnyxProps = SplitBillDetailsPageTransactionOnyxProps & SplitBillDetailsPageOnyxPropsWithoutTransaction;
```
and then
1st HOC
```
withOnyx<Omit<SplitBillDetailsPageProps, keyof SplitBillDetailsPageTransactionOnyxProps>, SplitBillDetailsPageOnyxPropsWithoutTransaction>
```
2nd HOC
```
withOnyx<SplitBillDetailsPageProps, SplitBillDetailsPageTransactionOnyxProps>
```